### PR TITLE
Bringing crucibles into reality

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Bugfixes "adjusts incorrect values for crucibles, obsolete recipe"```

#### Purpose of change
Crucible is made out of steel and as a 2L partially hollow item weights 14kg...

In general metalworking crucibles are made from graphite (density ~2.2g/cm3), also one simply cannot melt steel in crucible made out of steel - for obvious reasons.
Clay crucibles are perfectly fine, though they wouldn't last long in use. Fired clay density is in similar range (2-2.3g/cm3).

#### Describe the solution
Dividing the weights by 10 gives values close to expected from graphite/clay crucibles of given size, which leads me to believe that was the original intent and someone typo-ed extra digit - corrected
Obsoleted crucible recipe.
Changed material to ceramic.

#### Describe alternatives you've considered
Continuing to ignore this affronts to sanity.

#### Testing
Game loads correctly.

### CODE REVIEW

**Comment by Fris0uman** [comment]
Two questions: - We don't melt stuff that could melt in a steel crucible?

- there's no way to make a ceramic crucible?

**Comment by 1n17** [comment]
Sure, but that would be limited to lead, tin and its alloys, even bronze and brass would be too hot for steel crucible. But even giving it a little leeway - what are the recipes that use crucible yet don't require steel melting temperatures?

Making graphite/other ceramic crucible of sufficient quality is quite involved process, and for game purposes it would effectively just double clay crucible. I believe its best left as loot only, given them being slightly smaller and lighter.

[https://graphitemachining.cn/blog/2016/01/30/graphite-crucibles-production-process/](url)

**Comment by Fris0uman** [comment]
> what are the recipes that use crucible yet don't require steel melting temperatures?
I don't know, that was my question actually. If there is no recipe like that then it's all good, but if there is we might wan to introduce crucible quality and have low temp stuff require quality 1 ( steel crucible) and hight temp ones use quality 2 ( clay/cramic crucible)

**Comment by 1n17** [comment]
I see, well, there aren't so far. There seem to have been some preparation done for bronze work, but steel crucible is a no go for that. Steel loses tensile strength rapidly once approaching glowing temperatures and considering how easy it to make clay crucible both in game as IRL there would be no point in keeping steel crucible. Especially since for any purpose that it would be feasible for any decent thickness steel/cast iron pot would do as well.

File: data/json/recipes/recipe_others.json [hunk 1]
```diff
@@ -2087,19 +2087,6 @@
     "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "forge", 150 ], [ "oxy_torch", 30 ] ] ],
     "components": [ [ [ "steel_chunk", 160 ], [ "steel_lump", 40 ], [ "frame", 4 ], [ "hdframe", 2 ] ] ]
   },
-  {
-    "type": "recipe",
```
Comment (Inglonias) [inline data/json/recipes/recipe_others.json]: Is there going to be a craftable crucible to replace the recipe that was deleted here, or is this just going to be something you can only scavenge for?

**Comment by davidpwbrown** [comment]
Its true that clay crucible in-game is very easy to make, probably easier than the steel recipe, so its not too much of a hardship to remove the recipe for the steel one.

File: data/json/recipes/recipe_others.json (hunk 1 above)
Comment (1n17) [reply data/json/recipes/recipe_others.json]: No, crucible_clay is meant to be sole craftable crucible

**Comment by 1n17** [comment]
@davidpwbrown and give the player a reason to actually go look for clay

**Comment by Inglonias** [comment]
That's not a problem per se, but you mentioned that the clay crucibles probably wouldn't stand up to too much use. Are there future plans to represent that in gameplay terms, by giving crucibles a limited number of uses or something?

**Comment by 1n17** [comment]
No, that would require some infrastructure changes not warranted for single low value item, but might be worth a revisit once some other things that could break during crafting come up. At some point giving it a small chance to break at each use would be a nice touch.

**Comment by shadowsapex** [comment]
A clay crucible can't be made in a lab therefore metalworking would no longer be a feasible way to escape a lab. You're probably contributing to the death of lab escapes

**Comment by Inglonias** [comment]
> 
> 
> A clay crucible can't be made in a lab therefore metalworking would no longer be a feasible way to escape a lab. You're probably contributing to the death of lab escapes

The PR is based on the logic that you cannot use a steel container to heat steel to its melting point because the container will melt. If there's one thing I've learned about this project, it's that gameplay takes second stage to realism. Sometimes this is good, other times this is bad.

That being said, you can mod the recipe back into the game very easily,

**Comment by 1n17** [comment]
@shadowsapex indeed, but appropriate solution would be to add (uncraftable) crucible to some of lab drop list, it should be not an uncommon sight in a lab after all.

**Comment by shadowsapex** [comment]
> The PR is based on the logic that you cannot use a steel container to heat steel to its melting point because the container will melt. If there's one thing I've learned about this project, it's that gameplay takes second stage to realism. Sometimes this is good, other times this is bad.
> 
> That being said, you can mod the recipe back into the game very easily,

I am informing. Can we not do that now

> @shadowsapex indeed, but appropriate solution would be to add (uncraftable) crucible to some of lab drop list, it should be not an uncommon sight in a lab after all.

that doesn't really make sense

**Comment by I-am-Erk** [comment]
Adding a crucible to a lab makes perfect sense. They're common lab equipment. One of these days I plan on making a full on metallurgy lab, but crucibles are common in any chem lab.

**Comment by Maleclypse** [comment]
From wikipedia : "One of the earliest uses of platinum was to make crucibles." Finally a use for all those dental grilles!

**Comment by 1n17** [comment]
Note reads: BRB - Off to farm 1000+ platinum dental grills to make a platinum crucible.

**Pull request merged**